clients/src/main/java/org/apache/kafka/clients/admin/NewPartitionReassignment.java

```diff
@@ -18,16 +18,22 @@
 package org.apache.kafka.clients.admin;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 
 /**
  * A new partition reassignment, which can be applied via {@link AdminClient#alterPartitionReassignments(Map, AlterPartitionReassignmentsOptions)}.
  */
 public class NewPartitionReassignment {
     private final List<Integer> targetReplicas;
 
+    public static Optional<NewPartitionReassignment> of(Integer... brokers) {
+      return Optional.of(new NewPartitionReassignment(Arrays.asList(brokers)));
+    }
+
     public NewPartitionReassignment(List<Integer> targetReplicas) {
```

Comment:
Does this need to be public? Making it private forces use of `of` method, which I think is good.

Reply:
This makes a lot of sense to me :)